app/routes.py

```diff
@@ -1,2 +1,224 @@
-from flask import Blueprint
+from app import db, SLACK_TOKEN
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import request, Blueprint, make_response, jsonify
+from datetime import datetime
+import requests
 
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+def is_complete_helper_function(completed_at):
+    '''Takes in the completed_at attribute from Task, returns a boolean'''
+    if completed_at is None:
+        return False
+    else:
+        return True
+
+@tasks_bp.route("", methods = ["GET", "POST"])
+def handle_tasks():
+    if request.method == "GET":
+        sort_query = request.args.get("sort")
+        if sort_query == "asc":
+            tasks = Task.query.order_by(Task.title).all()
+        elif sort_query == "desc":
+            tasks = Task.query.order_by(Task.title.desc()).all()
+        else:
+            tasks = Task.query.all()
+        tasks_response = []
+        for task in tasks:
+            tasks_response.append({
+                "id": task.task_id,
+                "title": task.title,
+                "description": task.description,
+                "is_complete": is_complete_helper_function(task.completed_at)})
+        return jsonify(tasks_response)
+    elif request.method == "POST":
+        request_body = request.get_json()
+        if "title" not in request_body or "description" not in request_body or "completed_at" not in request_body:
+            return make_response({
+                "details": "Invalid data"
+                }, 400)
+        else:
+            new_task = Task(title=request_body["title"], description=request_body["description"], completed_at=request_body["completed_at"])
+            db.session.add(new_task)
+            db.session.commit()
+            return make_response(
+                    { "task": {
+                    "id": new_task.task_id,
+                    "title": new_task.title,
+                    "description": new_task.description,
+                    "is_complete": is_complete_helper_function(new_task.completed_at)
+                    }}, 201)
+            
+
+
+@tasks_bp.route("/<task_id>", methods = ["GET", "PUT", "DELETE"])
+def handle_task(task_id):
+    task = Task.query.get(task_id)
+    if task is None:
+            return make_response("", 404)
```

Comment:
this works great! we could shorten it by using another method:

```suggestion
    task = Task.query.get_or_404(task_id)
```